### DISCUSSION TITLE
Automated stowing of the arm rests in cockpit when pressed

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#667] [Ideas]
Instead having to tediously try to stow and put the arm rests in the cockpit away, maybe it could be changed so that when you click on one, it has a whole animation and does it by itself, kinda like in the flightfactor 757 for example. This would be much better and a QOL feature.

**Comment by aeroniemi** [comment]
it can be done fairly easily, although i wouldn't think people would want that as it's quite annoying in the 757

**Reply by Simleybob** [reply]
Would be much less annoying as the current method, thanks

**Reply by aeroniemi** [reply]
if people vote for wanting it then we can consider doing it
